test/DynamoCoreTests/WatchNodeTests.cs

```diff
@@ -98,17 +119,17 @@ public void WatchLiterals()
             ViewModel.Model.PreferenceSettings.NumberFormat = "f0";
 
             // get count node
-            Watch watchNumber = model.CurrentWorkspace.NodeFromWorkspace("eed0b6aa-0d82-44c5-aab6-2bf131044940") as Watch;
-            Watch watchBoolean = model.CurrentWorkspace.NodeFromWorkspace("8c5a87db-2d6a-4d3c-8c01-5ff326aef321") as Watch;
-            Watch watchPoint = model.CurrentWorkspace.NodeFromWorkspace("f1581148-9318-40fa-9402-61557255162a") as Watch;
+            var watchNumber = model.CurrentWorkspace.NodeFromWorkspace("eed0b6aa-0d82-44c5-aab6-2bf131044940") as Watch;
+            var watchBoolean = model.CurrentWorkspace.NodeFromWorkspace("8c5a87db-2d6a-4d3c-8c01-5ff326aef321") as Watch;
+            var watchPoint = model.CurrentWorkspace.NodeFromWorkspace("f1581148-9318-40fa-9402-61557255162a") as Watch;
 
-            WatchViewModel node = watchNumber.GetWatchNode();
+            var node = GetWatchViewModel(watchNumber);
             Assert.AreEqual("5", node.NodeLabel);
 
-            node = watchBoolean.GetWatchNode();
-            Assert.AreEqual("False", node.NodeLabel);
+            node = GetWatchViewModel(watchBoolean);
+            Assert.AreEqual("false", node.NodeLabel);
```

Comment:
Believe it or not, this was the sole mistake that relegate this to the Failure category.